CHANGES.md

```diff
@@ -78,6 +78,8 @@
 
 ### Fixed and improved
 
+* DC/OS checks: Update checks timeout
```

Comment:
This should be more descriptive, and it needs a ticket. And this is the same change as https://github.com/dcos/dcos/pull/2974, so they should add the same changelog entry.